namespace Kellox.Exceptions;

//Break exception (used to terminate a loop)
internal class Break : ApplicationException
{

}
